docs/DeveloperGuide.md

```diff
@@ -790,3 +790,29 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Effort**
+
+* As we were building off of AB3, we were able to reuse the model package and the existing commands.
+* In the beginning of our project (v1.2), this saved us time and effort as it was easy to build an MVP from an already
+functioning product (AB3) by making basic modifications to suit our product.
+* We were also able to reuse the extensive test cases and utility in the testutil package, which saved us a lot of
+effort. Future test cases and utility were also designed after these existing ones.
+* However, challenges arose when we wanted to refine our MVP features to better suit ResearchRoster (v1.3)
+and add new features unlike what AB3 had (v1.3, v1.4, v1.5).
+* This was because we had to carefully look through the AB3 code to fully understand how it worked,
+in order to figure out how we can modify it, work with it, or work around it to implement changes and new features.
+* This resulted in us putting in a high amount of effort to maintain good OOP and use/build off the existing classes
+and methods in AB3, rather than simply creating new classes, methods and solutions that may have overlapped with
+existing classes and methods.
+* The high effort was also because some of the existing classes methods posed limitations to what we wanted to 
+implement, and we had to come up with workarounds to achieve our desired modifications.
```

Comment:
Just one more thing - might be useful to talk about the export and assign commands which make the product all the more beneficial to researchers. Sorry I forgot about those earlier oops. 

Can just add another line after the edit command one about how we needed to add new command classes to create commands catering specifically to the needs of researchers

Reply:
right i forgot lol i think i spoke about assign a bit below but not explicitly